mailserver.env

```diff
@@ -13,11 +13,6 @@
 # => Specify a fully-qualified domainname to serve mail for.  This is used for many of the config features so if you can't set your hostname (e.g. you're in a container platform that doesn't let you) specify it in this environment variable.
 OVERRIDE_HOSTNAME=
 
-# (deprecated: use LOG_LEVEL instead)
-# 0 => Debug disabled
-# 1 => Enables debug on startup
-DMS_DEBUG=0
```

Comment:
Same here

Reply:
Done in 05f483554433eec6beadccc076a5432f8e262d38